### DISCUSSION TITLE
Octopus V2

### DISCUSSION THREAD

**Discussion by iplayfast** [ggml-org/llama.cpp#6526] [General]
https://arxiv.org/abs/2404.01744
https://huggingface.co/NexaAIDev/Octopus-v2

Personally I would like better than gtp-4 on my laptop :)

**Comment by lin72h** [comment]
I think the it's finetuned from gemma-2b, so there already support for it https://github.com/ggerganov/llama.cpp/pull/5631

**Reply by AndreasKunar** [reply]
FYI [impactframes/Octopus-v2-Q4_K_M-GGUF](https://huggingface.co/impactframes/Octopus-v2-Q4_K_M-GGUF/) seems to work. No need to download/convert yourself, like I did initially, because the 1st huggingface.co quantized GGUF I picked did not work.

Just as clarification, Octopus V2 is not a "normal" model like gemma-1.1-2b-it, its one specialized in function calling for agents.

**Comment by zhiyuan8** [comment]
Many thanks for your interest on Octopus model, we have also open-sourced V4 and published a quantized version

https://huggingface.co/NexaAIDev/Octopus-v4
https://huggingface.co/NexaAIDev/octopus-v4-gguf
